Meeting minutes 2024-08-29 (#81)
CPAN-Security/security.metacpan.org: meetings/cpansec-minutes-2024-08-29.md

## Meeting details

* August 29th, 2024 16:00 UTC on #cpansec-discussion on Matrix

### Welcome

*   Meeting chair: @sjn
*   Meeting secretary: All together

### Attendees, absents & regrets

*   Attendees
    *  @sjn, @timlegge, @stigtsp, @tux

### Approve previous meeting minutes
*   Previous meeting minutes was approved by @tux, @timlegge, @stigtsp

### Quick summary of current work
*   @tux - released DBI. CVE updates might be needed upstream
*   @tux - Ideas about distroprefs to be used for known CVE patches not integrated by maintainers: @tux will create a ticket and a short readme
*   @stigtsp - cpanm (cpanminus) vuln is kind off finished now, major distributions are following up already
*   @tux - Test::CVE is being used!

### Standards of Conduct
*   @sjn - TPRF has changed their SoC, should we follow suit? Discuss & decide.
*   Options are:
    1. https://github.com/tpf/soc/blob/main/Standards_of_Conduct.md
    2. https://openssf.org/community/code-of-conduct/
*   To update the current ticket prior to the next meeting
*   Vote at next meeting

### Vulnerability Index
*   @stigtsp - need to do some considerations on this vs. CNA

> *   the CNA requires a public index of CVEs - most open projects use github

Reviewer: ```suggestion
*   the CNA requires a public index of CVEs - some OSS projects use github for their index
```
